app/views/includes/result-item.nunjucks

```diff
@@ -9,7 +9,7 @@
   {% if gp.address.addressLines %} {{ gp.address.addressLines | safe | replace(",", ", ") }}, {% endif %}
   {% if gp.address.postcode %} {{ gp.address.postcode }} {% endif %}
 {% endif %}
-{% if gp.supplier %}
+{% if gp.bookingSystem %}
```

Comment:
Could a `not` be added here e.g. `{% if not gp.bookingSystem %}` to avoid the empty block?